### PR TITLE
Add account linking support

### PR DESCRIPTION
**Pull request by kirre-bylund**

### CODE REVIEW

File: LootLockerSDK/Source/LootLockerSDK/Private/GameAPI/LootLockerAccountLinkRequestHandler.cpp [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+{
+    HttpClient = NewObject<ULootLockerHttpClient>();
+}
+
+void ULootLockerAccountLinkRequestHandler::StartAccountLinkingProcess(const FLootLockerAccountLinkStartResponseBP& OnResponseCompletedBP, const FLootLockerAccountLinkStartResponseDelegate& OnResponseCompleted)
+{
+    LLAPI<FLootLockerAccountLinkStartResponse>::CallAPI(HttpClient, FLootLockerEmptyRequest{}, ULootLockerGameEndpoints::StartAccountLinkingProcess, { }, {}, OnResponseCompletedBP, OnResponseCompleted);
+}
+
+void ULootLockerAccountLinkRequestHandler::CheckStatusOfAccountLinkingProcess(const FString& LinkID, const FLootLockerAccountLinkProcessStatusResponseBP& OnResponseCompletedBP, const FLootLockerAccountLinkProcessStatusResponseDelegate& OnResponseCompleted)
```
Comment (JohannesLoot) [inline LootLockerSDK/Source/LootLockerSDK/Private/GameAPI/LootLockerAccountLinkRequestHandler.cpp:19]: I would rename this to "CheckAccountLinkingProcessStatus instead. Follows the same structure as the other functions then.

File: LootLockerSDK/Source/LootLockerSDK/Private/GameAPI/LootLockerAccountLinkRequestHandler.cpp [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+{
+    LLAPI<FLootLockerAccountLinkProcessStatusResponse>::CallAPI(HttpClient, FLootLockerEmptyRequest{}, ULootLockerGameEndpoints::CheckStatusOfAccountLinkingProcess, { LinkID }, {}, OnResponseCompletedBP, OnResponseCompleted);
+}
+
+void ULootLockerAccountLinkRequestHandler::CancelAccountLinkingProcess(const FString& LinkID, const FLootLockerCancelAccountLinkingProcessResponseBP& OnResponseCompletedBP, const FLootLockerCancelAccountLinkingProcessResponseDelegate& OnResponseCompleted)
+{
+    LLAPI<FLootLockerCancelAccountLinkingProcessResponse>::CallAPI(HttpClient, FLootLockerEmptyRequest{}, ULootLockerGameEndpoints::CancelAccountLinkingProcess, { LinkID }, {}, OnResponseCompletedBP, OnResponseCompleted);
+}
+
+void ULootLockerAccountLinkRequestHandler::UnlinkProviderFromAccount(const ELootLockerPlatform Provider, const FLootLockerUnlinkProviderFromAccountResponseBP& OnResponseCompletedBP, const FLootLockerUnlinkProviderFromAccountResponseDelegate& OnResponseCompleted)
```
Comment (JohannesLoot) [inline LootLockerSDK/Source/LootLockerSDK/Private/GameAPI/LootLockerAccountLinkRequestHandler.cpp:29]: UnlinkProviderFromAccount -> UnlinkProviderFromAccountLink

**Review by JohannesLoot — APPROVED** [review]
I know that it's not released yet, but so far so good!

File: LootLockerSDK/Source/LootLockerSDK/Public/LootLockerSDKManager.h [hunk 3]
```diff
@@ -330,6 +332,49 @@ class LOOTLOCKERSDK_API ULootLockerSDKManager : public UObject
      */
     static void WhiteLabelRequestPasswordReset(const FString& Email, const FLootLockerDefaultDelegate& OnCompletedRequest);
 
+    //==================================================
+	// Account Linking https://ref.lootlocker.com/game-api/#universal-auth
```
Comment (kirre-bylund) [inline LootLockerSDK/Source/LootLockerSDK/Public/LootLockerSDKManager.h:336]: Account Linking is a working title. So don't merge until this is resolved.

File: LootLockerSDK/Source/LootLockerSDK/Private/GameAPI/LootLockerAccountLinkRequestHandler.cpp (hunk 1 above)
Comment (kirre-bylund) [reply LootLockerSDK/Source/LootLockerSDK/Private/GameAPI/LootLockerAccountLinkRequestHandler.cpp]: Fixed

**Pull request merged**